dist/dist.py

```diff
@@ -2,21 +2,18 @@
 
 # source_files --
 #    Return a list of the WiredTiger source file names.
-def source_files(skip_includes=False):
-    if not skip_includes:
-        for line in glob.iglob('../src/include/*.[hi]'):
-            yield line
+def source_files():
     file_re = re.compile(r'^\w')
     for line in open('filelist', 'r'):
         if file_re.match(line):
-            yield os.path.join('..', line.rstrip())
+            yield os.path.join('..', line.split(None, 1)[0])
```

Comment:
I think these can just be `list.split()[0]` -- no need to pass anything to split?